### PR TITLE
Updates to signing and distribution methods

### PR DESCRIPTION
**Pull request by MonicaG**
Some minor updates to the documentation:

* [Apple app signing](https://dev.developer.gov.bc.ca/docs/default/component/mobile-developer-guide/apple_app_signing/)
  * Updated the secret key names. They were updated in the bcgov GitHub org to make the names clearer to their purpose. Updated the documentation to reflect that change
* [Distribution methods](https://dev.developer.gov.bc.ca/docs/default/component/mobile-developer-guide/distribution_methods/) 
  * Added in a sentence reflecting that both OCIO Mobile Device and Developer Experience teams need to take steps in setting up an internal app for distribution
  * Added headers to sections to help break up the content
* [Google app signing](https://dev.developer.gov.bc.ca/docs/default/component/mobile-developer-guide/google_app_signing/)
  * Added documentation on how to regenerate an upload key 

The linked pages contain the updates on the dev site.

### CODE REVIEW

File: docs/apple_app_signing.md [hunk 1]
```diff
@@ -39,12 +39,14 @@ To use a GitHub Actions to build and sign your app:
...
 1. We will give it access to the following secrets:
-  1. `BUILD_CERTIFICATE_BASE64`
-  1. `IOS_BUILD_CERTIFICATE_PASSWD`
+  1. `APPLE_APP_STORE_BUILD_CERTIFICATE_BASE64`
+  1. `APPLE_APP_STORE_BUILD_CERTIFICATE_PASSWD`
 1. Create and download the provisioning profile for your app
 1. Install the provisioning profile as a secret in your repo
   1. Refer to GitHub's [Installing an Apple certificate on macOS runners for Xcode development](https://docs.github.com/en/actions/deployment/deploying-xcode-applications/installing-an-apple-certificate-on-macos-runners-for-xcode-development) documentation for details
-1. In the Signing & Capabilities -> Release tab in Xcode turn off "automatically manage signing" 
+1. In the Signing & Capabilities -> Release tab in Xcode:
```
Comment (CoffeeGurlNia) [inline docs/apple_app_signing.md:47]: Replace ampersands with the full word "and" when possible. Ampersands can be challenging for the visually impaired, non-English speakers and those using assistive technologies to distinguish.

File: docs/apple_app_signing.md [hunk 2]
```diff
@@ -39,12 +39,14 @@ To use a GitHub Actions to build and sign your app:
...
+  1. Turn off "automatically manage signing" 
+  1. Select the new provisioning profile in the "Provisioning Profile" drop down box
```
Comment (CoffeeGurlNia) [inline docs/apple_app_signing.md:49]: Change "drop down" (verb) to "drop-down" (adjective).

**Review by CoffeeGurlNia — CHANGES_REQUESTED** [review]

File: docs/apple_app_signing.md (hunk 1 above)
Comment (MonicaG) [reply docs/apple_app_signing.md:47]: Xcode labels the tab as "Signing & Capabilities" (with the &). I copied the label from Xcode as that is what is displayed to the developer. I can change it though if it doesn't need to match

Comment (MonicaG) [reply docs/apple_app_signing.md:47]: Leaving as is, as it is how Xcode labels it

**Review by CoffeeGurlNia — APPROVED** [review]

**Pull request merged**